packages/frint-component-utils/src/streamProps.js

```diff
@@ -80,14 +84,14 @@ class Streamer {
         };
       });
 
-    this._push(Observable.of(object));
+    this._push(of$(object));
 
     return this;
   }
 
   get$() {
-    return Observable.merge(...this._observables)
-      .scan((props, emitted) => {
+    return merge$(...this._observables)
+      ::scan$((props, emitted) => {
```

Comment:
Subjective observation: adding `::` and `$` makes code look weird. But I see where the need comes from: glueing things together and indicating methods returning observables.

Reply:
yep, initially I wanted to avoid this `$` sign. but then I realized we import modules from both RxJS and Lodash.

imagine the `merge` module which is common in both `rxjs` and `lodash`. maintaining the convention of always ending  variable/function name with `$` helps avoid any clash:

```js
import merge from 'lodash/merge';
import { merge as merge$ } from 'rxjs/observable/merge';
```

Reply:
yes, totally see why you did that